Makefile

```diff
@@ -4,7 +4,7 @@ all: docs/es/all.yaml docs/index.yaml
 RELEASE := $(shell awk '$$1 == "version:"{v=$$2} $$1 == "name:"{n=$$2} END {print n "-" v}' enterprise-suite/Chart.yaml )
 
 docs/es/all.yaml: docs/$(RELEASE).tgz
-	helm install $< --name=es --namespace=lightbend --dry-run --debug | sed -e '1,/^---$$/ d' > $@
+	helm install $< --namespace=lightbend --dry-run --debug | sed -e '1,/^---$$/ d' > $@
```

Comment:
Even though the "helm install" has dry-run specified, this line would fail if the 'es' release happened to be installed on a running minikube.  The name value shows up only in the section removed by the following sed expression, so we can use the default name and not worry about the state of the minikube.